app/controllers/tasks_controller.rb

```diff
@@ -10,6 +10,8 @@ def index
   end
 
   def show
+    @page_title = @task.title
+    @last_editor = last_editor(@task)
```

Comment:
Out of curiosity, what is suppose to happen when `Task.find(params[:id])` can not find a task?

Reply:
There would be nothing to show. @task itself is set by the authorization framework. If there is no task, the view will have all kinds of issues.
Here is the def of `last_editor`
https://github.com/ReadyResponder/ReadyResponder/blob/b38950c24f73ae3075107b3cf9d4c26122f2d266/app/controllers/application_controller.rb#L19

Reply:
I put my comment one line too low.

I thought @task was set by the `before_action :set_task`. Is it suppose to be set by the authorization framework? That does not appear to be what is happening
  
```
    def set_task
      @task = Task.find(params[:id])
    end
```